docs/pages/guides/tutorials/wordpress-mysql-persistentvolume.md

```diff
@@ -0,0 +1,232 @@
+
+    ![Create secret component](./screenshots/wp8.png)
+
+2. Click on the Secret component to open the configuration window.
+
+    - Set the _name_ as `mysql-pass`
+    - Set the _Type_ as `Opaque`.
+    - Click **+** next to Data and add the secret as a key value pair `password:cGFzc3dvcmQ=`
+
+    ![Configure secret](./screenshots/wp9.png)
+
+3. Click outside the window to close the configuration tab.
+
+#### Create Persistent Volumes
+
+MySQL and WordPress each require a [Persistent Volume(PV)](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) to store their data.
+
+For this tutorial, we will use the `manual` StorageClassName and set the Persistent Volume to use the `hostPath` type.
+
+Please note that using `hostPath` for Persistent Volumes is generally not recommended for production environments because it ties the volume to the node's filesystem, which can lead to data loss if the node fails. However, we can use it in this tutorial for development purposes.
+
+1. Click on the Kubernetes icon on the dock, search for `Persistent Volume`, and select it. We will create two PV's.
+
+    ![Create persistent volume](./screenshots/wp10.png)
+
+2. Click on the wordpress PV to open the configuration window.
+
+    - Change the "name" to `wp-pv`
+    - Set the "StorageClassName" as `manual`
+    - Click **+** next to "AccessMode" and enter `ReadWriteOnce`
+
+    ![Configure persistent volume](./screenshots/wp11.png)
+
+    - Scroll down to "Capacity" and enter in the key pair `storage:20Gi`
+
+    ![Persistent volume capacity](./screenshots/wp12.png)
+
+    - Scroll down to "Hostpath" and input `mnt/data/wp-pv` for the _path_ and `DirectoryOrCreate` for the _type_.
+
+    ![Persistent volume hostpath](./screenshots/wp13.png)
+
+3. Repeat similar steps for the Mysql Persistent Volume
+
+    - Click on the MySQL PV to open the configuration window.
+    - Change the "name" to `mysql-pv`
+    - Set the "StorageClassName" to `manual`
+    - Click **+** next to "AccessMode" and set it to `ReadWriteOnce`
+    - Scroll down to "Capacity", and enter the key pair `storage:20Gi`
+    - Scroll down to "Hostpath" and input `mnt/data/mysql-pv` for the _path_ and `DirectoryOrCreate` for the _type_.
+
+4. Click on `wp-pv-claim` and `mysql-pv-claim` and set their "StorageClassName" as `manual`
+
+#### Add Labels
+
+To make it easier to filter and manage our resources during visualization, let's add labels to all of them.
+
+ You can also choose to use the existing label, `app:wordpress`, but a new one, `dev:tutorial`, is recommended for this tutorial to prevent our resources from getting mixed up with others in the public playground cluster.
+
+1. Click on the label icon.
+2. Click the **+** sign next to Labels.
+3. Add the label `dev:tutorial`.
+4. Do this for **all** the resources on the canvas.
+
+    ![Add label](./screenshots/wp14.png)
+
+5. Now click on `Save As` and save the design.
```

Comment:
Is saving the design necessary? Or is there an intent behind this?
All changes done after importing the YAML should be saved already isn't it.

Reply:
It was to give it a new name after merging.